.github/workflows/lint.yml

```
@@ -0,0 +1,16 @@
+name: "Lint"
+on:
+  push:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Worth switching to newer v4
```suggestion
        uses: actions/checkout@v4
```